src/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixer.php

```diff
@@ -130,7 +131,7 @@ public function isRisky()
     /**
      * {@inheritdoc}
      */
-    protected function applyFix(\SplFileInfo $file, Tokens $tokens)
```

Comment:
Please revert unrelated changes. They should be applied globally in dedicated PRs.

Reply:
This must have been my muscle memory on the spot, you say it totally right this change has nothing at all to do with the issue at hand and has been removed from the change-set.

Please consider this comment addressed.